main.pas

```diff
@@ -4201,12 +4206,19 @@ procedure TForm1.FormShow(Sender: TObject);
     if mylang = 2 then
       PikaGetFile(flp, 'spa.txt', path + 'config.ppk', 'Build By Schthack');
 
-    if dragenabled = 1 then
+    if dragenabled then
       smDrag.Checked := true
```

Comment:
smDrag.Checked := dragenabled;

since they are both boolean

other whay too that wasnt supported in delphi 7
when you have an integer you can do something like smDrag.Checked := intValue = 1;
this will convert to true when intValue is 1